cypress/support/fragments/settings/remote-storage/configurations.js

```diff
@@ -308,4 +311,18 @@ export default {
   confirmCreateRemoteStorage() {
     cy.do(confirmationModal.find(saveBtn).click());
   },
+  opentEditConfigurationForm(name) {
+    cy.do([
+      MultiColumnListCell({ content: name }).click(),
+      Pane({ title: name }).find(actionsBtn).click(),
+      Button({ id: 'clickable-edit-storage' }).click(),
```

Comment:
let's move repeated locators in the file into const variables

Reply:
done